platform_versioned_docs/version-23.4/getting-started/rnaseq.mdx

```diff
@@ -0,0 +1,752 @@
+
+- The **Input form view** displays form fields to enter text, select attributes from dropdowns, and browse input and output locations with [Data Explorer](../data/data-explorer.mdx).
+- The **Config view** displays raw configuration text that you can edit directly. Select JSON or YAML format from the **View as** dropdown.
+- **Upload params file** allows you to upload a JSON or YAML file with run parameters.
+
+Platform uses the `nextflow_schema.json` file in the root of the pipeline repository to dynamically create a form with the necessary pipeline parameters. 
+
+Specify your input and output parameters:
+
+<details>
+  <summary>**input**</summary>
+
+  Use **Browse** to select your pipeline input data: 
+
+  - In the **Data Explorer** tab, select the existing cloud bucket that contains your samplesheet, browse or search for the samplesheet file, and select the chain icon to copy the file path before closing the data selection window and pasting the file path in the input field.
+  - In the **Datasets** tab, search for and select your existing dataset.
+
+  ![Input parameters](./quickstart-demo/assets/sp-cloud-launch-parameters-input.gif)
+
+</details>
+<details>
+  <summary>**outdir**</summary>
+
+  Use the `outdir` parameter to specify where the final results created by the pipeline are published. `outdir` must be unique for each pipeline run. Otherwise, your results will be overwritten. 
+
+  Select **Browse** to specify a cloud storage directory using Data Explorer, or enter a cloud storage directory path to publish pipeline results to manually.
+
+  ![Output parameters](./quickstart-demo/assets/sp-cloud-launch-parameters-outdir.gif)
+
+</details>
+
+Modify other parameters to customize the pipeline execution through the parameters form. For example, under **Read trimming options**, change the `trimmer` to select `fastp` in the dropdown menu instead of `trimgalore`.
+
+![Read trimming options](./quickstart-demo/assets/trimmer-settings.png)
+
+### Advanced settings 
+
+- Use [resource labels](../resource-labels/overview.mdx) to tag the computing resources created during the workflow execution. While resource labels for the run are inherited from the compute environment and pipeline, admins can override them from the launch form. Applied resource label names must be unique. 
+- [Pipeline secrets](../secrets/overview.mdx) are used to store keys and tokens used by workflow tasks to interact with external systems. Enter the names of any stored user or workspace secrets required for the workflow execution.
+- See [Advanced options](../launch/advanced.mdx) for more details.
+
+After you have filled the necessary launch details, select **Launch**. The **Runs** tab shows your new run in a **submitted** status at the top of the list. Select the run name to navigate to the [**View Workflow Run**](../monitoring/overview.mdx) page and view the configuration, parameters, status of individual tasks, and run report.
+
+<details>
```

Comment:
Something weird happening here. It is shown in the right side navigation but won't take me here when I click on it. Is a good idea to have these heading hidden like this?

Reply:
E.g., view reports does nothing when you click on it.

Reply:
You're right. I changed the config for the page so that only H2s are shown on the ToC on the right. So the headings inside that collapsed section are now excluded from the ToC.